scraper/dev/docker/Dockerfile.base

```diff
@@ -52,8 +52,10 @@ COPY Pipfile .
 COPY Pipfile.lock .
 
 ENV LANG en_US.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
 ENV PIPENV_HIDE_EMOJIS 1
 RUN apt-get update -y && apt-get install -yq \
-    python-pip
-RUN pip install pipenv
-RUN pipenv install
\ No newline at end of file
+    python3-pip
```

Comment:
this is unrelated to the commit and to the PR, please move to another

Reply:
This is required by the PR since otherwise we do have the package import issue due to twisted

Reply:
What about putting it in a separate commit?